extensions/wikia/AdEngine3/src/setup.js

```diff
@@ -179,6 +179,13 @@ function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = tru
 		if (!isGeoEnabled('wgAdDriverPubMaticOutstreamCountries')) {
 			context.remove('bidders.prebid.pubmatic.slots.INCONTENT_PLAYER');
 		}
+
+		const unbiddedSlotName = hasFeaturedVideo ? 'INCONTENT_PLAYER' : 'FEATURED';
+		let videoBidders = ['appnexusAst', 'beachfront', 'lkqd', 'pubmatic', 'rubicon'];
```

Comment:
Can we mark bidders as "video" bidders in `ad-context.js` and draw the list from there? Or iterate over bidder configs checking if they include definition for slots icp or featured? This way we will have a single source of truth.

Reply:
Great idea, changed.